server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const port = 3000;
+
+app.use(express.static('dist'));
+
+app.get('*', (req, res) => {
+  res.sendFile(path.resolve(__dirname, 'dist', 'index.html'));
+});
+
+app.listen(port, () => {
```

Comment:
Может быть такое, что порт 3000 занят другим приложением, было бы классно дать возможность задавать порт через команду запуска nodejs скрипта

```suggestion
app.listen(process.env.PORT || PORT, () => {
```